src/ui/components/NetworkMonitor/index.tsx

```diff
@@ -62,13 +66,16 @@ export const NetworkMonitor = ({
   }, [container.current]);
 
   if (loading) {
+    mixpanel.time_event("netMonitor.open_network_monitor");
```

Comment:
Ooooo interesting - I haven't seen this before. Do we need to guard against situations where tracking is disabled?